.editorconfig

```diff
@@ -0,0 +1,18 @@
+# .editorconfig file
```

Comment:
[curious] Does this file need to be in here?

Reply:
It doesn't _need_ to be, no. However, defining the `.editorconfig` file will help IDEs ensure consistent file formatting. For example, without this file, whenever I create a new file within VSCode, it defaults to using tabs (4 characters), requiring manual changes to change tabs to 2 spaces.

The `.editorconfig` files dictates to VSCode to set those params on new files on its own, for example.